examples/devtools/intercept.js

```diff
@@ -1,11 +1,11 @@
 /**
- * in order to run this file make sure you have `webdriverio` and `devtools`
```

Comment:
This example should also work without using the devtools protocol to run the test in. With adding the network interceptor functionalities we now initialise Puppeteer also when running local Chromedriver tests.

Reply:
Let's create another example for webdriver protocol?

Reply:
it would be the same file just with Chromedriver running in the background. I just tested it and found that we should also change:

```patch
+ const puppeteerBrowser = await browser.getPuppeteer()
- const puppeteerBrowser = browser.getPuppeteer()
```

as well as:

```patch
    browser = await remote({
-       automationProtocol: 'devtools',
        capabilities: {
            browserName: 'chrome'
        }
    })
```

Reply:
applied the suggested changes